### DISCUSSION TITLE
Is the Slingbox M2 not compatible?

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#131] [General]
Just curious if the compatibility for the M1 includes the M2 or is it that the M2 is completely incompatible?

**Comment by GerryDazoo** [comment]
M2 works. Just use the 350/500/M1 SBTYPE

**Reply by GR0DZ1LLA** [reply]
Ok, that's what I'm testing.  Thanks.
